dict/dict0dict.cc

```diff
@@ -433,20 +428,20 @@ bool dict_table_col_in_clustered_key(const dict_table_t *table, ulint n) {
 void dict_init() {
   dict_sys = (dict_sys_t *)mem_alloc(sizeof(dict_sys_t));
 
-  mutex_create(&dict_sys->mutex, IF_DEBUG("dict_mutex",) IF_SYNC_DEBUG(SYNC_DICT,) Source_location{});
+  mutex_create(&dict_sys->mutex, IF_DEBUG("dict_mutex", ) IF_SYNC_DEBUG(SYNC_DICT, ) Source_location{});
+  dict_sys->table_hash = new std::unordered_map<std::string, dict_table_t *>();
```

Comment:
Can we make this into a std::string_view? It should be a pointer to dict_table_t->name. std::string will use more memory.